### PR TITLE
Implement accept, acceptRun func and refactoring state functions

### LINKED ISSUE #56: [lex] Implement accept function in lexer
**Subject**
lex.go

**Detail**
 Implement `accept` function

**Reference**
go-sdk1.11
src/text/template/parse/lex.go

### PR DESCRIPTION
**Pull request by joomanzi**
resolved: #56  #57 

details:
1. Implement accept and acceptRun w/ testcase
2. refactoring numberStateFn and spaceStateFn
 - isSpace and isDIgit can be replaced w/ accept function

- [x] Test case

### CODE REVIEW

File: parse/lex.go [hunk 1]
```diff
@@ -304,30 +307,18 @@ func identifierStateFn(s *state, e emitter) stateFn {
...
 		e.emit(errToken)
 		return defaultStateFn
 	}
 
-	for isSpace(s.peek()) {
-		s.next()
-	}
-
 	//absorb..
+	s.acceptRun(" \t")
```
Comment (junbeomlee) [inline parse/lex.go]: how about define space string? like " ", "\t", etc
`spaceChars = " \t"`

Comment (junbeomlee) [reply parse/lex.go]: Except this LGTM

Comment (joomanzi) [reply parse/lex.go]: agree

**Review by junbeomlee — APPROVED** [review]
Good

**Pull request merged**